content/frontend/vue-apollo/8-subscriptions.md

```diff
@@ -264,6 +266,8 @@ Open `src/components/LinkList.vue` and add the following object to the `sunscrib
 {
   document: NEW_VOTES_SUBSCRIPTION,
   updateQuery: (previous, { subscriptionData }) => {
+  	if (!subscriptionData.data.Vote) return
```

Comment:
Remove spaces at the start so it align with `const` on the next line (indentation)